templates/triton_services/README.ipynb

```diff
@@ -0,0 +1,334 @@
+    ")\n",
+    "triton_server.start(wait_until_ready=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compile the model using Triton's Python backend. This will take 10-15 minutes on a\n",
+    "T4 GPU and 8-10 minutes on an A10 GPU. The model will be compiled and saved in the\n",
+    "`model_repository` directory as TensorRT engine. Also keep in mind the model has to be\n",
+    "built in the same type of GPU you are planning to serve the model on. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "import datetime\n",
+    "\n",
+    "\n",
+    "print(f\"start time: {datetime.datetime.now()}\")\n",
+    "t0 = time.time()\n",
+    "model = triton_server.load(\"stable_diffusion_1_5\")\n",
+    "duration = time.time() - t0\n",
+    "print(f\"Total duration: {duration}s\")\n",
+    "\n",
+    "# Unload the model and the server to free the memory.\n",
+    "triton_server.unload(model, wait_until_unloaded=True)\n",
+    "triton_server.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After the model is compiled, we can upload the model to a cloud storage. We need to\n",
+    "upload both the model config file `config.pbtxt` and the TensorRT engine model\n",
+    "directory. Anyscale provides a environment variable `ANYSCALE_ARTIFACT_STORAGE` that\n",
+    "can be used to store model artifacts. Use one of the following to upload the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# If running in AWS\n",
+    "!aws s3 cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/config.pbtxt $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/config.pbtxt\n",
+    "!aws s3 cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ --recursive"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If running in GCP\n",
+    "!gcloud storage cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/config.pbtxt $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/config.pbtxt\n",
+    "!gcloud storage cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ --recursive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Triton Server locally on Anyscale Workspaces\n",
+    "\n",
+    "The `Deployment.py` is included in this workspace for you. This file demonstrates using\n",
```

Comment:
it's lowercase "deployment.py"

Reply:
also let's name this something more specific please like `triton_app.py` otherwise it's confusing with the serve term "deployment"